internal/project/init.go

```diff
@@ -0,0 +1,102 @@
+/*
+ * Flow CLI
+ *
+ * Copyright 2019-2021 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package project
+
+import (
+	"bytes"
+	"fmt"
+	"text/tabwriter"
+
+	"github.com/onflow/flow-cli/pkg/flow/util"
+
+	"github.com/onflow/flow-cli/internal/command"
+
+	"github.com/onflow/flow-cli/pkg/flow"
+
+	"github.com/onflow/flow-cli/pkg/flow/services"
+	"github.com/spf13/cobra"
+)
+
+type flagsInit struct {
+	ServicePrivateKey  string `flag:"service-priv-key" info:"Service account private key"`
```

Comment:
I just realized that the `accounts add` command uses `private-key` instead of `priv-key`. Maybe we can standardize?

Reply:
Already fixed that yesterday myself